edx_proctoring/templates/proctoring/seq_proctored_exam_instructions.html

```diff
@@ -0,0 +1,125 @@
+{% load i18n %}
+<div class="sequence proctored-exam instructions" data-exam-id="{{exam_id}}" data-exam-started-poll-url="{{exam_started_poll_url}}">
+  <h3>
+    {% blocktrans %}
+      Awaiting Proctoring Installation & Set Up
+    {% endblocktrans %}
+  </h3>
+  <p>
+    {% blocktrans %}
+      You've chosen to take {{display_name}} as a proctored exam. You should be directed to a new window
```

Comment:
@chrisndodge I've never experienced a redirect to the system check window. If this is not as designed, we should probably reword the text.